src/wp-includes/class-wp-query.php

```diff
@@ -797,9 +797,8 @@ public function fill_query_vars( $query_vars ) {
 	 *                                                   disable cache priming for term meta, so that each
 	 *                                                   get_term_meta() call will hit the database.
 	 *                                                   Defaults to the value of `$update_post_term_cache`.
-	 *     @type bool            $lazy_load_post_meta    Whether to lazy-load post meta. Setting to false will
-	 *                                                   disable cache priming for post meta, so that each
-	 *                                                   get_post_meta() call will hit the database.
+	 *     @type bool            $lazy_load_post_meta    Whether to lazy-load post meta. If set to false, post meta priming will
+	 *                                                   fallback to the default behavior of priming post meta.
```

Comment:
I think the following changes better reflect the current state of the logic and resolves conversations [here](https://github.com/WordPress/wordpress-develop/pull/4216/files/434cb55fca1d6d86d3f230256f02ac9333943992..34855d127635aa49606cbd81293dbad8b2c616e9#r1878491731) and [here](https://github.com/WordPress/wordpress-develop/pull/4216/files/434cb55fca1d6d86d3f230256f02ac9333943992..34855d127635aa49606cbd81293dbad8b2c616e9#r1878494301).

```suggestion
	 *     @type bool            $lazy_load_post_meta    Whether to lazy-load post meta. When set to true, post meta priming will
	 *                                                   be disabled. Default false.
```